client/angular.json

```diff
@@ -136,7 +136,7 @@
       },
       "root": "apps/admin-portal",
       "sourceRoot": "apps/admin-portal/src",
-      "prefix": "app",
+      "prefix": "sb",
```

Comment:
If you are trying to change the `npmScope`, this is not where it is being handled.

Do it in the `nx.json` file under `"npmScope: "app"` change it to: `"npmScope: "sb"`

Reply:
Also, go through and switch all the prefixes on the components from `app-` to `sb-`.

Reply:
Also, update all the "`paths`" from `tsconfig.json`.